apps/data-service-catalog/middleware.ts

```diff
@@ -29,11 +29,11 @@ export default withAuth(
     ];
 
     if (catalogId && !validOrganizationNumber(catalogId)) {
-      return NextResponse.rewrite(new URL('/notfound', process.env.NEXTAUTH_URL));
+      return NextResponse.rewrite(new URL('/catalogs/notfound', process.env.NEXTAUTH_URL));
```

Comment:
Not found blir vel brukt for alle sider og dermed skal våre på root

Reply:
74a44dbe0d1491649bc24cfa0ff4d3c4057bef91